src/test/java/htsjdk/variant/variantcontext/GenotypeLikelihoodsUnitTest.java

```diff
@@ -82,6 +82,13 @@ public void testFromString2() {
         Assert.assertEquals(gl.getAsString(), vPLString);
     }
 
+    @Test
+    public void testFromString3() {
```

Comment:
please explicitly test what would happen in the case that some of the elements are missing and others not (so that there's an explicit example of lost data.